### DISCUSSION TITLE
UI Meeting

### DISCUSSION THREAD

**Discussion by charlieikosi** [pinpointltdpng/ODFD-PROJECT#14] [Announcements]
Hi @anapitalai @sylvery

I would like to set a meeting for tomorrow 12pm to go through the UI.

Please let me know if you will be available.

Thanks

**Comment by anapitalai** [comment]
I am available at that time.

On Tue, Apr 11, 2023 at 9:10 AM charlieikosi ***@***.***>
wrote:

> Hi @anapitalai <https://github.com/anapitalai> @sylvery
> <https://github.com/sylvery>
>
> I would like to set a meeting for tomorrow 12pm to go through the UI.
>
> Please let me know if you will be available.
>
> Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pinpointltdpng/ODFD-PROJECT/discussions/14>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABUMTLPLDODAKUZIWQ4LG5DXASHNNANCNFSM6AAAAAAWZQWFGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

Alois Ignatius Napitalai
IT Lab Manager
Surveying and Land Studies Department
PNG University of Technology
PMB 411
Lae
Phone: 473 4963

https://www.google.com/+AloisNapitalai
https://www.facebook.com/alois.napitalai
https://www.facebook.com/NAPITALAIICT
***@***.***
<http://www.unitechblogspot.wordpress.com>

**Reply by charlieikosi** [reply]
Awesome!

**Comment by sylvery** [comment]
I'll be free

**Reply by charlieikosi** [reply]
Perfect!
